heron/schedulers/src/java/com/twitter/heron/scheduler/aurora/AuroraCLIController.java

```diff
@@ -112,6 +112,9 @@ public void removeContainers(Set<PackingPlan.ContainerPlan> containersToRemove)
     }
   }
 
+  private static final String ERR_PROMPT =
```

Comment:
This error message is declared and used in a few different places. Is it possible to clean it up?

Reply:
this file in schedulers directory, the others are in scheduler-core. what do you suggest?